src/Containers/Auth/AuthReducers.ts

```diff
@@ -0,0 +1,50 @@
+import { AuthState } from './AuthTypes';
+import { AuthActions } from './AuthActions';
+
+export const initialState: AuthState = {
+	identity: {
+    authorizedAt: null,
+		accessToken: null,
+		refreshToken: null,
+		expiresIn: null
+	},
+	me: null
+}
+
+const AuthReducer = (
+	state: AuthState = initialState,
+  action: AuthActions,
+): AuthState => {
+	switch(action.type) {
+    case 'UPDATE_IDENTITY':
+			let newState = Object.assign({}, state);
```

Comment:
this can be short as `return {...state, identity: {...payload, authorizedAt: new Date()}};`
Anyway don't use Date() in reducer, it was a side effect and make function un-pure. Init the date before dispatch action instead.